src/ui.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Container,
+  render,
+  Text,
+  VerticalSpace,Code,Space
+} from "@create-figma-plugin/ui";
+import { h } from "preact";
+import { useEffect } from "preact/hooks";
+import {copyToClipboard} from 'figx'
```

Comment:
**Nitpick**: The lack of white spaces makes me think that it's time to introduce husky and the new Styleint 😊